types/stake.go

```diff
@@ -42,8 +42,9 @@ type Validator interface {
 	GetOperator() ValAddress      // operator address to receive/return validators coins
 	GetConsPubKey() crypto.PubKey // validation consensus pubkey
 	GetConsAddr() ConsAddress     // validation consensus address
-	GetPower() Int                // validation power
 	GetTokens() Int               // validation tokens
+	GetBondedTokens() Int         // validator bonded tokens
+	GetTendermintPower() int64    // validation power in tendermint
```

Comment:
Do we really need this on the SDK type?

Reply:
I will verify this

Reply:
It is used in genesis, I think it's reasonable to keep this property